### ISSUE TITLE
pytorch is less accurate than mxnet on LSTM regression model

### ISSUE THREAD

**Issue by horsepurve** [pytorch/pytorch#3066]
Hi, I implement a LSTM regression model for sequence labeling using both MXNet and PyTorch with the same network parameters, however, the precision of PyTorch is less than MXNet, though it is ~6 times faster than MXnet:

MXNet: Pearson correlation of prediction: 0.99168
PyTorch: Pearson correlation of prediction: 0.98714

this precision is important for me for publication, so does anyone have any suggestion? 

Thanks!

**Comment by soumith** [comment]
we cant help without seeing more details. It is likely that you added something around your loss function (or part of the network) in a numerically unstable way.
Can you open a thread at https://discuss.pytorch.org with more details.
